### DISCUSSION TITLE
IBAX ÖNCE NEREDE LİSTELENECEK ?

### DISCUSSION THREAD

**Discussion by cancan80** [IBAX-io/go-ibax#1081] [Soru Cevap - Türkçe]
IBAX ÖNCE NEREDE LİSTELENECEK?

**Comment by Skips1** [comment]
Çok yeni kanka şuan hiçbir şey belli değil

**Comment by Yusufknk** [comment]
Coinmarketcap ve Coingecko

**Reply by CODOMAN** [reply]
Binance ile anlaştı diye biliyorum ama emin değilim duyumlar öyle tewetırdaki yabancılar öyle yazıyor.

**Reply by 2558Mete** [reply]
Binance harika olur

**Comment by CODOMAN** [comment]
> IBAX ÖNCE NEREDE LİSTELENECEK?

Binance diye anlaştılar diyen yabancılar var twetırda ama onlarında bir kısmı öyle diyor bazıları huobi diyor.

**Comment by erhanqwerty** [comment]
binance olabilir.

**Comment by sedat2855** [comment]
BİNANCE MUHTESEM OLUR
